target/linux/ath79/dts/qca9533_tplink_cpe210.dtsi

```diff
@@ -4,20 +4,9 @@
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
```

Comment:
You can remove those includes, as they are already in the parent DTSI.

Reply:
I will remove this